### PR TITLE
Add value name to bindings

### PR DESCRIPTION
**Pull request by slamberg94**
Makes ffi declarations less fragile to refactoring.

### CODE REVIEW

File: src/Navigation.re [hunk 1]
```diff
@@ -37,10 +37,10 @@ external addListener:
...
+[@bs.send] external pop: t => unit = "pop";
+[@bs.send] external popN: (t, int) => unit = "popN";
+[@bs.send] external popToTop: t => unit = "popToTop";
 // Drawer navigator only
-[@bs.send] external openDrawer: t => unit = "";
-[@bs.send] external closeDrawer: t => unit = "";
-[@bs.send] external toggleDrawer: t => unit = "";
+[@bs.send] external openDrawer: t => unit = "openDrawer";
+[@bs.send] external closeDrawer: t => unit = "closeDrawer";
+[@bs.send] external toggleDrawer: t => unit = "toggleDrawer";
```
Comment (MoOx) [inline src/Navigation.re]: @cknitt Is refmt removing newlines or is your editor doing that?

**Comment by MoOx** [comment]
@slamberg94 Is refmt removing newlines or is your editor doing that?

**Comment by Natim** [comment]
@MoOx his editor. I just checked out the branch and ran `yarn format` and it fixed it.

**Comment by Natim** [comment]
```
diff --git a/src/Navigation.re b/src/Navigation.re
index 0d440f7..b67ef5a 100644
--- a/src/Navigation.re
+++ b/src/Navigation.re
@@ -43,4 +43,4 @@ external addListener:
 // Drawer navigator only
 [@bs.send] external openDrawer: t => unit = "openDrawer";
 [@bs.send] external closeDrawer: t => unit = "closeDrawer";
-[@bs.send] external toggleDrawer: t => unit = "toggleDrawer";
\ No newline at end of file
+[@bs.send] external toggleDrawer: t => unit = "toggleDrawer";
diff --git a/src/NavigationActions.re b/src/NavigationActions.re
index 4751b06..79a9c2e 100644
--- a/src/NavigationActions.re
+++ b/src/NavigationActions.re
@@ -18,4 +18,4 @@ external backParams:
 external navigate: navigateParams => action = "navigate";
 
 [@bs.module "react-navigation"] [@bs.scope "NavigationActions"]
-external back: backParams => action = "back";
\ No newline at end of file
+external back: backParams => action = "back";
diff --git a/src/NavigationContainer.re b/src/NavigationContainer.re
index 9bc5d89..55229f0 100644
--- a/src/NavigationContainer.re
+++ b/src/NavigationContainer.re
@@ -1,3 +1,3 @@
 type t;
 [@bs.send]
-external dispatch: (t, NavigationActions.action) => unit = "dispatch";
\ No newline at end of file
+external dispatch: (t, NavigationActions.action) => unit = "dispatch";
```

File: src/Navigation.re (hunk 1 above)
Comment (MoOx) [inline src/Navigation.re]:
```suggestion
[@bs.send] external toggleDrawer: t => unit = "toggleDrawer";

```

File: src/NavigationActions.re [hunk 2]
```diff
@@ -15,7 +15,7 @@ external backParams:
   "";
 
 [@bs.module "react-navigation"] [@bs.scope "NavigationActions"]
-external navigate: navigateParams => action = "";
+external navigate: navigateParams => action = "navigate";
 
 [@bs.module "react-navigation"] [@bs.scope "NavigationActions"]
-external back: backParams => action = "";
+external back: backParams => action = "back";
```
Comment (MoOx) [inline src/NavigationActions.re]:
```suggestion
external back: backParams => action = "back";

```

File: src/NavigationContainer.re [hunk 3]
```diff
@@ -1,2 +1,3 @@
 type t;
-[@bs.send] external dispatch: (t, NavigationActions.action) => unit = "";
+[@bs.send]
+external dispatch: (t, NavigationActions.action) => unit = "dispatch";
```
Comment (MoOx) [inline src/NavigationContainer.re]:
```suggestion
external dispatch: (t, NavigationActions.action) => unit = "dispatch";

```

**Pull request closed without merging**